include/seqan3/range/container/bitcompressed_vector.hpp

```diff
@@ -964,37 +977,37 @@ class bitcompressed_vector
     //!\brief Checks whether `*this` is equal to `rhs`.
     constexpr bool operator==(bitcompressed_vector const & rhs) const noexcept
     {
-        return data == rhs.data;
+        return raw_data() == rhs.raw_data();
```

Comment:
Are these changes necessary? I don't care much ....

Reply:
Mhm, I was afraid of the code coverage since I didn't want to test the `data`/`raw_data` functions.

I'll revert it and see what happens.